### PR TITLE
Sonarqube

### PR DESCRIPTION
**Pull request by mcbloch**

### CODE REVIEW

File: host_files/herbert/nginx/sonarqube.zeus.gent [hunk 1]
```diff
@@ -0,0 +1,37 @@
+upstream cat {
+   server 10.1.0.210:9000;
+   keepalive 32;
+}
+
+server {
+    listen 80;
+    server_name cat.zeus.gent;
```
Comment (Jan-PieterBaert) [inline host_files/herbert/nginx/sonarqube.zeus.gent]: are you sure this shouldn't be sonarqube.zeus.gent

File: host_files/herbert/nginx/sonarqube.zeus.gent [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    if ($request_method = GET) {
+        return 301 https://$server_name$request_uri;
+    }
+
+    return 308 https://$server_name$request_uri;
+}
+
+server {
+    listen 443 ssl http2;
+    server_name cat.zeus.gent;
```
Comment (Jan-PieterBaert) [inline host_files/herbert/nginx/sonarqube.zeus.gent]: are you sure this shouldn't be sonarqube.zeus.gent

Comment (mcbloch) [reply host_files/herbert/nginx/sonarqube.zeus.gent]: Good point ;)

File: host_files/herbert/nginx/sonarqube.zeus.gent [hunk 3]
```diff
@@ -0,0 +1,37 @@
+upstream cat {
+   server 10.1.0.210:9000;
+   keepalive 32;
+}
```
Comment (fbegyn) [inline host_files/herbert/nginx/sonarqube.zeus.gent]: Pretty sure this should also be changed to `sonarcube` then, and then maybe double check of the upstream IP is correct

File: host_files/herbert/nginx/sonarqube.zeus.gent [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+       client_max_body_size 50M;
+       proxy_buffers 256 16k;
+       proxy_buffer_size 16k;
+       client_body_timeout 60;
+       send_timeout 300;
+       lingering_timeout 5;
+       proxy_connect_timeout 90;
+       proxy_send_timeout 300;
+       proxy_read_timeout 90s;
+       proxy_pass http://cat;
```
Comment (fbegyn) [inline host_files/herbert/nginx/sonarqube.zeus.gent]:
```suggestion
       proxy_pass http://sonarqube;
```
Forgot to change this and now I'm doing it the right way

Comment (mcbloch) [reply host_files/herbert/nginx/sonarqube.zeus.gent]: Nice, I did a quick copy and paste and obviously missed most of the stuff I had to change :|

**Pull request merged**